Assets/HoloToolkit/SpatialSound/Scripts/AudioOccluder.cs

```diff
@@ -9,47 +9,62 @@ namespace HoloToolkit.Unity
     /// <summary>
     /// Class that implements IAudioInfluencer to provide an occlusion effect.
     /// </summary>
-    public class AudioOccluder : MonoBehaviour, IAudioInfluencer
+    /// <remarks>
+    /// For best results, ensure that all sound emitting objects have an attached
+    /// AudioInfluencerManager. Failing to do so may result in incorrect frequency
+    /// and/or volume restoration when the RemoveEffect method is called.
+    /// </remarks>
+    public class AudioOccluder 
+        : MonoBehaviour, 
+        IAudioInfluencer
     {
         /// <summary>
-        /// Frequency above the nominal range of human hearing. Applying this frequency to the filter will have no perceived impact on the audio source.
+        /// Frequency above which sound will not be heard after applying occlusion.
+        /// Setting this value to 22000.0 effectively disables the effect.
         /// </summary>
-        private readonly float NeutralFrequency = 22000f;
-
-        [Tooltip("Frequency above which sound will not be heard.")]
+        /// <remarks>
+        /// Chaining occluders will result in the lowest of the cutoff frequencies being applied to the sound.
+        /// The CutoffFrequency range is 0.0 - 22000.0 (0 - 22kHz), inclusive.
+        /// The default value is 5000.0 (5kHz).
+        /// </remarks>
+        [Tooltip("Frequency above which sound will not be heard after applying occlusion.")]
         [Range(10.0f, 22000.0f)]
         public float CutoffFrequency = 5000.0f;
-        
+
+        /// <summary>
+        /// Percentage of the audio source volume that will be heard after applying occlusion.
+        /// </summary>
+        /// <remarks>
+        /// VolumePassThrough is cumulative. It is applied to the current volume of the object at the time
+        /// the effect is applied.
+        /// The VolumePassThrough range is from 0.0 - 1.0 (0-100%), inclusive.
+        /// The default value is 1.0.
+        /// </remarks>
         [Tooltip("Percentage of the audio source volume that will be heard after applying occlusion.")]
         [Range(0.0f, 1.0f)]
         public float VolumePassThrough = 1.0f;
 
-        // Update is not used, but is kept so that this component can be enabled/disabled.
-        private void Update() 
```

Comment:
have you tested that this component can now be enabled / disabled as per the comment.

Reply:
@SimonDarksideJ it was moved to the end of the file for some reason.  We can probably revert this change.

Reply:
It got moved to get it out of the way. I tried eliminating it and was no longer able to enable/disable the component.

@SimonDarksideJ, yes, I have tested (and verified via Unity docs) that Update is one of the functions that allows for enabling/disabling components.

Reply:
So from your testing, this update call has to stay in or consumers can't enable / disable it on the fly

Reply:
Yes. Per the documentation for MonoBehaviour, one or more of the following must be present to be able to enable/disable on the fly.

Note: There is a checkbox for disabling MonoBehaviour on the Unity Editor. It disables functions when unticked. If none of these functions are present in the script, the Editor does not display the checkbox. The functions are:
Start()
Update()
FixedUpdate()
LateUpdate()
OnGUI()
OnDisable()
OnEnable()

Update is convenient, but then again, so is Start().  I am willing to have an empty method for either of those two.